Sources/Kitura/resources/index.html

```diff
@@ -1,153 +1,46 @@
-                            </div>
-                        </div>
-                    </div>
-                    <div class="contentTrailingRepo">
-                        <img src="/@@Kitura-router@@/git-logo.png"/>
-                        <div class="contentSectionTextWrapper">
-                            <a href="https://github.com/IBM-Swift/KituraKit" class="contentSectionTitle">IBM-Swift/KituraKit</a>
-                            <div class="contentSectionText">
-                                Client side library for REST requests using Swift 4's Codable protocol.
-                            </div>
-                        </div>
-                    </div>
-                    <div class="contentTrailingRepo">
-                        <img src="/@@Kitura-router@@/git-logo.png"/>
-                        <div class="contentSectionTextWrapper">
-                            <a href="https://github.com/IBM-Swift/Swift-Kuery-ORM" class="contentSectionTitle">IBM-Swift/Swift-Kuery-ORM</a>
-                            <div class="contentSectionText">
-                                A Codable Object Relational Mapping (ORM) library for Swift.
-                            </div>
-                        </div>
-                    </div>
-                </div>
-            </div>
-        </div>
-
-        <div class="footerBar">
-            <div class="footerText" style="margin-left: 40px;">
-                &copy; Copyright IBM Corporation 2017. Licensed under the Apache License, Version 2.0.
-            </div>
-            <a class="footerSwift" href="https://developer.ibm.com/swift/">Swift@IBM</a>
-        </div>
-    </div>
-</body>
+    <link rel="stylesheet" href="/@@Kitura-router@@/main.css">
+    <link href=“https://fonts.googleapis.com/css?family=IBM+Plex+Sans” rel=“stylesheet”>
+    <link rel="icon" type="image/png" href="/@@Kitura-router@@/favicon.png" sizes="16x16" />
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+  </head>
+  <body>
+    <section class="main-section">
+      <section class="top-section">
+        <img src="/@@Kitura-router@@/kitura-bird.svg" alt="">
+        <h1>Your Kitura server is up and running!</h1>
+        <section class="button-section">
+          <button class="button" type="button" name="Kitura.io button" onclick="window.open('http://kitura.io/')"><img class="kitura-image" src="/@@Kitura-router@@/white-kitura.svg" alt="White Kitura logo">KITURA.IO</button>
+          <button class="button" type="button" name="Slack button" onclick="window.open('http://slack.kitura.io/')"><img class="slack-image" src="/@@Kitura-router@@/white-slack.svg" alt="White Slack logo">SLACK</button>
+          <button class="button" type="button" name="API button" onclick="window.open('https://ibm-swift.github.io/Kitura/')"><img class="api-image" src="/@@Kitura-router@@/api-icon.png" alt="Gear cog icon">KITURA API</button>
```

Comment:
API DOCS like on kitura.io

Reply:
You want this to link out to the entire API docs rather than just the Kitura API docs?